### PR TITLE
[infra] Update exclude.txt

### PR DESCRIPTION
**Pull request by hseok-oh**
This commit updates execlude.txt for code quality analysis.

ONE-DCO-1.0-Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

### CODE REVIEW

**Comment by hseok-oh** [comment]
This change will be useful on different repo name.

File: .ahub/sam/exclude.txt [hunk 1]
```diff
@@ -1,53 +1,49 @@
 # External code: Android NN API
-/ONE/compiler/ann-api/include/NeuralNetworks.h
```
Comment (chunseoklee) [inline .ahub/sam/exclude.txt:2]: This works for /nnfw/XXX ?

**Review by chunseoklee — APPROVED** [review]
LGTM if it work for public tizen w/o change.

File: .ahub/sam/exclude.txt (hunk 1 above)
Comment (hseok-oh) [reply .ahub/sam/exclude.txt:2]: I checked this updates for ONE only and it worked.

**Pull request merged**